packages/sdks/src/blocks/img/img.lite.tsx

```diff
@@ -47,18 +50,34 @@ export default function ImgComponent(props: ImgProps) {
 
       return getSrcSet(url);
     },
+    get aspectRatioCss():
+      | (Pick<JSX.CSS, 'position' | 'height' | 'width' | 'left' | 'top'> & {
+          position: 'absolute';
+        })
+      | undefined {
+      const aspectRatioStyles = {
+        position: 'absolute',
+        height: '100%',
+        width: '100%',
+        left: '0px',
+        top: '0px',
+      } as const;
+      const out = props.aspectRatio ? aspectRatioStyles : undefined;
+      return out;
+    },
```

Comment:
is it possible to add an e2e test to test all of the inputs, that they're working as expected?

you can copy one of the Image spec example and instead of Image use Raw:Img